model/dependencies.go

```diff
@@ -80,6 +81,12 @@ func (di *dependencyIncluder) handle(pair TVPair) (bool, error) {
 		di.included[pair] = false
 		return false, errors.Errorf("task '%s' in variant '%s' cannot be run for a '%s'", pair.TaskName, pair.Variant, di.requester)
 	}
+
+	if utility.FromBoolPtr(bvt.Disable) {
+		di.included[pair] = false
+		return false, errors.Errorf("task '%s' in variant '%s' has been disabled", pair.TaskName, pair.Variant)
```

Comment:
did you test with dependencies? I'm not sure that we want this to be an error, we just don't want to include it. (We want tasks that depend on a disabled task to be scheduled, just without that dependency)

Reply:
Yeah this might be okay, we'll just need to test this.